mllib/src/main/scala/org/apache/spark/ml/classification/MultilayerPerceptronClassifier.scala

```diff
@@ -88,7 +88,7 @@ private[ml] trait MultilayerPerceptronParams extends PredictorParams
    * @group expertParam
    */
   final val weights: Param[Vector] = new Param[Vector](this, "weights",
-    " Sets the weights of the model ")
+    "Weights (either initial if before training or actual on model)")
```

Comment:
Perhaps we can clarify: `Model weight vector. If set before training, this represents the weight vector used to initialize training. If set on a trained model, this explicitly sets the model weights (overriding any existing weights)`, or something along those lines. Scala doc could be updated accordingly (agree it is unclear currently).